### PR TITLE
feat: Add Live Coding Sessions

### PR DESCRIPTION
**Pull request by leohhhn**
## Description

This PR adds the Gno Live Coding Session folder. This folder will host all future live coding sessions, along with their code, READMEs, video links, and possibly also other contributor submissions.

### CODE REVIEW

File: lcs/README.md
Comment (moul) [inline lcs/README.md:1]: i understand why you prefer "lcs" over "live-coding-session" but i think it makes sense to be more explicit while discovering the file structure.

maybe we can find something in between.

File: lcs/lcs-1/README.md
Comment (moul) [inline lcs/lcs-1/README.md:1]: We can keep a number as a prefix, such as 1-, 001-, or 050-, to make it sortable and allow for the addition of new, simpler sessions. Alternatively, we can use numbers as a unique identifier (probably as you did), which is also acceptable. However, I suggest using a small word other than "lcs," such as "amm," to differentiate the identifier.

`1-amm`?

File: lcs/lcs-1/README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Gno Live Coding Session #1
+
+#### Topic: Minimal Constant Product GRC20 AMM in Gno
+#### Date: Friday, Aug 16th 2024
+#### Presenter: [@leohhhn](https://github.com/leohhhn)
+
+This live coding session was the first Gno Live coding session.
+It included building a basic [`GRC20`](https://gno.land/p/demo/grc/grc20) DEX in Gno with the constant product AMM
```
Comment (moul) [inline lcs/lcs-1/README.md:8]: can you add a link to this algorithm, like a whitepaper, or maybe a solidity implementation?

File: lcs/README.md
Comment (leohhhn) [reply lcs/README.md:1]: Sure. I'd say we go with `live-coding` for explicitness.

File: lcs/lcs-1/README.md
Comment (leohhhn) [reply lcs/lcs-1/README.md:1]: sgtm

**Pull request merged**